src/components/Api.js

```diff
@@ -0,0 +1,104 @@
+export default class Api {
```

Comment:
It’s better to move api class to utils directory, since it's not a component, since it's not a component but a utility class